versioned/storage/bigtable/src/intTest/java/org/projectnessie/versioned/storage/bigtable/ITBigTableBackendFactory.java

```diff
@@ -84,6 +86,7 @@ public void productionLike() throws Exception {
 
           RepositoryLogic repositoryLogic = repositoryLogic(persist);
           repositoryLogic.initialize("initializeAgain");
+          RepositoryDescription repoDesc = repositoryLogic.fetchRepositoryDescription();
           soft.assertThat(repositoryLogic.fetchRepositoryDescription()).isEqualTo(repoDesc);
```

Comment:
Aren't we altering the test's logic? It used to compare `RepositoryDescription` returned by different `Backend` / `Persist` instances, now it's comparing output of two calls on the same instance.

If it's not critical, why bother with two separate `try` blocks?

Reply:
Whoops my bad, I did not look into what the assertion was trying to assert.